osgeo_importer/forms.py

```diff
@@ -41,6 +41,9 @@ def clean(self):
             if is_zipfile(f):
                 with ZipFile(f) as zip:
                     for zipname in zip.namelist():
+                        # Skip mac directory sidecar
+                        if zipname == '__MACOSX/':
+                            continue
```

Comment:
Not sure if there's a smarter way to do this, but in the end this seemed to be the 'simplest' way - just ignore this directory which is always present in files that were compressed in OS X.

I don't like that it's kind of ad hoc, but I haven't come up with a better solution. Open to suggestions.